resources/us_verifications/models/lob_confidence_score.yml

```diff
@@ -25,7 +26,21 @@ properties:
       - ""
     description: |
       Indicates the likelihood that the address is a valid, mail-receiving address. Possible values are:
-        - high — Over 70% of mailpieces Lob has sent to this address were delivered successfully.
+        - High — Over 70% of mailpieces Lob has sent to this address were delivered successfully and recent mailings were also successful.
```

Comment:
smh you just HAD to make it capitalized here when the others weren't. s m h @Thexumaker

Reply:
this is why I shouldn't do a pr at night LOL